hci.go

```diff
@@ -113,6 +113,7 @@ type leAdvertisingReport struct {
 
 type leConnectData struct {
 	connected      bool
+	disconnected   bool
```

Comment:
Why do we need two bools here? `connected` and `disconnected`.

Reply:
Yes, that is correct. Otherwise would get called more than once per disconnect.